sdk/src/main/java/com/sygic/travel/sdk/auth/facade/AuthFacade.kt

```diff
@@ -0,0 +1,128 @@
+package com.sygic.travel.sdk.auth.facade
+
+import android.util.Log
+import com.sygic.travel.sdk.auth.model.AuthorizationRequest
+import com.sygic.travel.sdk.auth.model.AuthorizationResponse
+import com.sygic.travel.sdk.auth.model.UserRegistrationResponse
+import com.sygic.travel.sdk.auth.service.AuthService
+import ru.gildor.coroutines.retrofit.Result
+import ru.gildor.coroutines.retrofit.awaitResult
+
+
+class AuthFacade(
+	private val authService: AuthService
+) {
+
+	suspend fun getUserSession(): AuthorizationResponse? {
+		val authRequest = AuthorizationRequest()
+		authRequest.clientId = CLIENT_ID
+		authRequest.grantType = GrantType.CLIENT_CREDENTIALS.grantType
```

Comment:
Not sure if enum of GrantType is better than constants.

Reply:
I cant also imagine we have no constants for this type.

Reply:
So should I use constants ?

Reply:
Personally, in this case I would put the string here directly.

Reply:
:heavy_check_mark: